apierr/error_overrides.go

```diff
@@ -33,4 +33,6 @@ var allTransientErrors = []*regexp.Regexp{
 	regexp.MustCompile(`Unknown worker environment`),
 	regexp.MustCompile(`ClusterNotReadyException`),
 	regexp.MustCompile(`worker env .* not found`),
+	regexp.MustCompile(`Timed out after `),
```

Comment:
You need to add these to `openapi/errors.go`, otherwise they will be removed during autogeneration.

Reply:
🤦 fixed...